ocs_ci/utility/azure_utils.py

```diff
@@ -39,7 +40,20 @@ def load_cluster_resource_group(cluster_path, terraform_filename=TERRRAFORM_FILE
     Returns:
         string with resource group name
     """
-    filepath = os.path.join(cluster_path, terraform_filename)
+    terraform_files = [
```

Comment:
I would also maybe do here:
if not terraform_filename:
and move rest of the code only to that block.

Reply:
updated!